atd-vze/src/views/Crashes/crashDataMap.js

```diff
@@ -92,18 +92,15 @@ export const crashDataMap = [
   {
     title: "Fatalities/Injuries",
     fields: {
-      apd_confirmed_fatality: {
-        label: "Fatality Flag",
-        editable: true,
-        uiType: "select",
-        lookupOptions: "atd_txdot__y_n_lkp",
-        lookupPrefix: "y_n",
```

Comment:
you can leave the `lookupOptions` and the `lookupPrefix`. This just enables the field to display as `YES` or `NO` instead of `Y` or `N`. And keeping those would be consistent with the rest of the fields on the "Flags" card section.

Reply:
Oops - good catch! I replaced those options and now we have `YES` or `NO` again.

![Screen Shot 2019-10-31 at 2 20 07 PM](https://user-images.githubusercontent.com/37249039/67979199-c7d3a980-fbe9-11e9-933d-f964915ac37d.png)